admin-ui/src/components/settings/settings-navigation.tsx

```
@@ -42,26 +40,18 @@ export function SettingsNavigation({
             type="button"
             onClick={() => onSectionClick(section.id)}
             className={cn(
-              "w-full text-left px-3 py-2 rounded-md text-sm transition-colors",
-              "hover:bg-accent hover:text-accent-foreground",
-              "focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring",
-              isActive && "bg-accent/50",
+              "relative w-full text-left px-3 py-2 rounded-sm text-[0.72rem] font-semibold uppercase tracking-[0.2em] transition-colors border border-transparent",
+              "hover:bg-accent/10 hover:border-border/60",
+              "focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring/40",
+              isActive &&
```

ESLint

[error] Delete `·&&`

(prettier/prettier)